pkgs/misc/emulators/caprice32/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, libpng, pkgconfig, SDL, freetype, zlib, mesa }:
+
+stdenv.mkDerivation rec {
+
+  repo = "caprice32";
+  version = "unstable-2018-02-10";
+  rev = "53de69543300f81af85df32cbd21bb5c68cab61e";
+  name = "${repo}-${version}";
+
+  src = fetchFromGitHub {
+    inherit rev repo;
```

Comment:
What was the reason to define ```repo``` and ```rev``` outside and then inherit here?
Why not just here?

Reply:
'cause i like this, it could be implicit rules.